turbinia/evidence.py

```diff
@@ -355,3 +356,44 @@ def __init__(self, artifact_name):
     super(ExportedFileArtifact, self).__init__()
     self.artifact_name = artifact_name
     self.copyable = True
+
+
+class DockerContainer(Evidence):
+  """Evidence object for a DockerContainer filesystem.
+
+  Attributes:
+    container_id(str): The ID of the container to mount.
+  """
+
+  def __init__(self, container_id=None, *args, **kwargs):
+    """Initialization for Docker Container."""
+    super(DockerContainer, self).__init__(*args, **kwargs)
+    self.container_id = container_id
+    self._container_fs_path = None
+    self._docker_root_directory = None
+    self._mount_path = None
+    self.context_dependent = True
+
+  def _preprocess(self):
+    self._mount_path = self.parent_evidence.local_path
```

Comment:
Though this can't happen if self.context_dependent is True, when instantiating a new DockerContainer evidence, I guess I can raise and explain why 'this shouldn't happen'

Reply:
Yeah, I think that's a good idea just as a catchall.  Some of this code is starting to get a bit complex, so it would be nice to have an explicit error just in case something changes down the line.